packages/web/src/components/PaperCard.tsx

```
@@ -1,72 +1,145 @@
+              !isExpanded && "line-clamp-3 max-h-[4.5em] overflow-hidden"
+            )}
+          >
+             {paper.abstract}
+             {!isExpanded && (
+               <div className="absolute bottom-0 left-0 w-full h-8 bg-gradient-to-t from-[var(--color-surface)] to-transparent" />
+             )}
+          </div>
+          <button
+            onClick={() => setIsExpanded(!isExpanded)}
+            className="mt-1 flex items-center gap-1 text-xs font-medium text-[var(--color-primary)] hover:underline focus:outline-none"
+          >
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)